File: src/main/java/com/ensa/jibi/domain/dto/ConfirmationPaiementDto.java
package com.ensa.jibi.domain.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ConfirmationPaiementDto {
    private Long id;
    private Double montant;
    private Long compteId;
    private Long creanceId;
    private LocalDate date;
}

File: src/app/entities/confirmation-paiement-dto.ts
import {CreanceDto} from "./creance-dto";
import {ComptePaiementDto} from "./comptePaiement-dto";

export interface ConfirmationPaiementDto {
  id: number;
  montant: number;
  compte: ComptePaiementDto;
  creance: CreanceDto;
  date: string;
}
